### PR TITLE
Package.json changes for coreclr debugging

### PR DESCRIPTION
**Pull request by caslan**

### CODE REVIEW

File: tools/InstallToVSCode/coreclr/package.json [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+  "description": "coreclr-debug",
+  "version": "0.1.0",
+  "publisher": "Microsoft",
+  "engines": {
+    "vscode": "^0.10.6"
+  },
+  "categories": [
+    "Debuggers"
+  ],
+  "activationEvents": [
```
Comment (gregg-miskelly) [inline tools/InstallToVSCode/coreclr/package.json]: I think you don't want this part

File: tools/InstallToVSCode/coreclr/package.json [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
+    "onLanguage:csharp",
+    "onLanguage:fsharp"
+  ],
+  "main": "./out/src/extension",
```
Comment (gregg-miskelly) [inline tools/InstallToVSCode/coreclr/package.json]: Not sure if this will cause any problems

File: tools/InstallToVSCode/coreclr/package.json [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
+  "scripts": {
+    "vscode:prepublish": "node ./node_modules/vscode/bin/compile",
+    "compile": "node ./node_modules/vscode/bin/compile -watch -p ./",
+    "postinstall": "node ./node_modules/vscode/bin/install"
+  },
```
Comment (gregg-miskelly) [inline tools/InstallToVSCode/coreclr/package.json]: Same

File: tools/InstallToVSCode/coreclr/package.json [hunk 4]
```diff
@@ -42,7 +61,94 @@
                 "description": "If true, the debugger should stop at the entry point of the target.",
                 "default": false
               },
-              // --- Begin common attributes. Copy these down to 'attach' if modified
```
Comment (gregg-miskelly) [inline tools/InstallToVSCode/coreclr/package.json]: We seem to have lost the 'common attributes' comment.

**Comment by gregg-miskelly** [comment]
Signing off with comments

**Comment by caslan** [comment]
Ok, I was trying to keep 2 copies of files in sync. Created another pull request with just the additions

**Pull request closed without merging**